src/utils/hact/raise-repair-form.js

```diff
@@ -31,17 +37,59 @@ export const buildRaiseRepairFormData = (formData) => {
         ],
       },
     ],
-    sitePropertyUnit: [
-      {
-        address: {
-          addressLine: [formData.shortAddress],
+    site: {
+      property: [
+        {
+          propertyReference: formData.propertyReference,
+          address: {
+            addressLine: [formData.shortAddress],
+          },
+          reference: [
+            {
+              id: formData.propertyReference,
+            },
+          ],
         },
+      ],
+    },
+    instructedBy: {
+      name: 'Hackney Housing',
+    },
+    assignedToPrimary: {
+      name: `Contractor ${formData.sorCodeContractorRef}`,
+      organization: {
         reference: [
           {
-            id: formData.propertyReference,
+            id: formData.sorCodeContractorRef,
           },
         ],
       },
-    ],
+    },
+    customer: {
+      name: formData.callerName,
+      person: {
+        name: {
+          full: formData.callerName,
+        },
```

Comment:
Not sure if this is the right thing here.. 
The customer could be different to the contact details that the operative provides on the raise repair form. Would this info come from the tenancy/resident API instead ?